src/test/java/com/cegeka/academy/repository/InterestTest.java

```diff
@@ -22,16 +24,59 @@
     @Autowired
     private UserRepository userRepository;
 
-    @Test
-    public void testAddInterest(){
-        Interest interest1  = new Interest();
+    private User user;
+
+    private Interest interest1;
+
+    private Interest interest2;
+
+    private Set<Interest> interestSet;
+
+    private Set<User> userSet;
+
+    @BeforeEach
+    public void setUp() {
+        userRepository.deleteAll();
+
+        user = new User();
+        user.setEmail("gigi.gogaie@gmail.com");
+        user.setFirstName("Gogaie");
+        user.setLastName("Momaie");
+        user.setPassword("aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa");
+        user.setLogin("gigel@purcel.com");
+
+        interest1  = new Interest();
```

Comment:
Pentru setUp as recomanda sa ai o metoda de creare a obiectelor de tip interest, iar atunci cand apelezi interestRepository.save(), vei apela metoda nou creată. Vezi in testul pentru UserChallenge. Mi-a cerut Catalin modificarea asta.